### ISSUE TITLE
Error on Homebridge 1.3.0

### ISSUE THREAD

**Issue by AlexNoop** [dotsam/homebridge-milight#74]
Hello, i have this error on homebridge 1.3.0 :

[2/21/2021, 6:27:26 PM] [homebridge-milight] This plugin generated a warning from the characteristic 'Color Temperature': characteristic was supplied illegal value: number 140 exceeded minimum of 153. See https://git.io/JtMGR for more info.

My config :

{
            "platform": "MiLight",
            "name": "MiLight",
            "bridges": [
                {
                    "ip_address": "192.168.0.24",
                    "version": "v6",
                    "use8Zone": true,
                    "lights": {
                        "fullColor": [
                            "Salon",
                            "Entrée",
                            "Cuisine",
                            "WC",
                            "Chambre",
                            "Chevets",
                            "Salle de Bain",
                            "Balcon"
                        ],
                        "bridge": [
                            "Box"
                        ]
                    }
                }
            ],
            "repeat": 1,
            "delay": 100
        }

Any idea ?

Thanks everybody ;)

**Comment by AlexNoop** [comment]
for info, same in 1.3.1

**Comment by yvesdecloedt** [comment]
same issue here. It started after upgrading my node.js to v14.16

**Comment by composerkaumann** [comment]
Because this plugin seems to be abandoned I had to switch to Homebridge Milight V6 which works with nodejs 15 and hb 131

**Comment by yvesdecloedt** [comment]
unfortunately the Homebridge Milight V6 does not work with the 'white' led controller. At least there is no selection for 'white' and the other types do not seem to work. 
I am a bit stuck. I am using node-red as main platform. The available milight plugins in node-red are very unstable (sometimes they work/sometimes they don't) and I used homebridge (milight v1.1.5) + integration in node-red for my led strips.

**Comment by dotsam** [comment]
So this warning is the opposite of the sort of warning HomeBridge is actually trying to warn about. In this case, I've set the min/max of the colour temperature characteristic to match the stated range of the MiLight bulbs. My assumption was that HomeKit apps would read the characteristic's mix/max values and not send values outside of that, but that doesn't seem to be the case. So the HomeKit app is sending a lower colour temperature value, HomeBridge is checking that agains the mix/max I have set, and is throwing this error.

I'll check if there's a different way to do what I'm trying to achieve, or if I just need to normalize the value in my code. But for now, this shouldn't cause any errors at the HAP protocol level, it's simply HomeBridge showing this warning, and shouldn't impact any functionality.
